### ISSUE TITLE
of-web-ui loads translation file from a non-existing directory (ui/assets/ instead of assets/ )

### ISSUE THREAD

**Issue by JeroenGommans** [opfab/operatorfabric-core#4879]
Using SNAPHOT (24-08)

When I enter the `config/docker` directory and execute the `startOpfab.sh` command, the UI is unable to load the translation files and I see menus named "menu.feed" and "menu.archives"

Looking at the of-web-ui logs, I see the following error:

`[error] 30#30: *29 open() "/usr/share/nginx/html/ui/assets/i18n/fr.json" failed (2: No such file or directory)`

When opening a bash shell in the UI image, I notice the assets directory is located directory under the 'html' directory, not under `ui`. Executing the following commands fixes the issue:

`mkdir /usr/share/nginx/html/ui`
`ln -s /usr/share/nginx/html/assets /usr/share/nginx/html/ui/assets`

**Comment by quinarygio** [comment]
Hello, 
I'm not able to reproduce it. 
In fact the location of ui assets has changed (see migration_guide_to_4.0.adoc)
Could you please try to rebuild the docker images (gradlew clean dockerTagSnapshot ) ?

**Comment by JeroenGommans** [comment]
Hi,

Thanks for looking into it.
I did not build Opfab myself, but used the docker images. 

I performed the following steps:
( I first removed all images from docker using `docker system prune -a` )
- created a new directory and clone the opfab repro to it. Branch is `develop`
- cd config/docker
- ./startOpfab.sh

When I now go to the http://localhost:2002, I see 'menu.feed' and 'menu.archives'
